content/v2.0/influxdb-templates/create.md

```diff
@@ -7,7 +7,7 @@ menu:
     parent: InfluxDB templates
     name: Create a template
     identifier: Create an InfluxDB template
-weight: 101
+weight: 102
 v2.0/tags: [templates]
 ---
 
```

Comment:
To create an InfluxDB template, do one of the following:
- [Import a template](...)
- [Create a template from an existing dashboard](...)
- [Clone a template](...)

Reply:
These items don't apply to the template creation workflow.